README.md

```diff
@@ -1 +1,3 @@
 # wn
+
+## what is this tool doing?
```

Comment:
Maybe we can add a note where the tool is hosted, what it is doing?

Reply:
You're right! I've been a little busy making changes to the interface but expect this to look a little less sad in a day or two :)